src/main/resources/org/sagebionetworks/web/client/view/SettingsViewImpl.ui.xml

```diff
@@ -12,40 +12,43 @@
     <bh:Div>
       <b:Row>
         <b:Column size="LG_6">
-          <bh:Div ui:field="editProfilePanel" addStyleNames="highlight-box">
```

Comment:
Remove usage of `highlight-box` in this file and just use divs with `highlight-title` for consistency.